xscen/config.py

```diff
@@ -108,6 +111,16 @@ def recursive_update(d, other):
     return d
 
 
+def args_as_str(*args: Tuple[Any, ...]) -> Tuple[str, ...]:
```

Comment:
If the goal is to allow passing a `Path` to `DataCatalog`, wouldn't returning a tuple break it?

Reply:
I'm fairly certain that the data type for a function using `*args` is `Tuple[Any, ...]`, hence why accessing `arg` from `*args` is typically done within a loop/indexing. I might be wrong, though.

Reply:
Right, that's indeed the case.